src/feditest/nodedrivers/imp/__init__.py

```diff
@@ -62,13 +62,7 @@ def http(self, request: HttpRequest, follow_redirects: bool = True) -> HttpReque
 
 
     # @override # from WebFingerClient
-    def perform_webfinger_query(
-        self,
-        resource_uri: str,
-        rels: list[str] | None = None,
-        check_content_type: bool = False,
-        validate_jrd: bool = False,
```

Comment:
Note that these defaulted to `False`, which allowed the tests to do test-specific assertions even if there were other issues like wrong content type or invalid JRD ("invalid" from the strict perspective of implementation-specific, non-standard extra keys, for example). I don't mind removing those keyword args, but I think we should keep the Imp behavior the same as it is now or accept test results which are less useful, in my opinion (which relates back to the Imp-as-fixture discussion).